packages/app/src/compiler/pipelines/vision-classify.base.json

```diff
@@ -0,0 +1,35 @@
+{
+  "plugins": {
+    "dataCollect": {
+      "package": "./packages/plugins/data-collect/image-classification-data-collect",
+      "params": {
+        "url": "<TODO>"
```

Comment:
nlp-classify.baes.json has the message 'input your CSV zip' and here the message is 'TODO'. This seems not consistent